src/service/bill_service.rs

```diff
@@ -218,6 +220,7 @@ pub struct BillService {
     store: Arc<dyn BillStoreApi>,
     identity_store: Arc<dyn IdentityStoreApi>,
     file_upload_store: Arc<dyn FileUploadStoreApi>,
+    network_kind: Network,
```

Comment:
Since we have the `Network` as a global now, we don't really need to pass it to the services - you can just use the global

Reply:
Adding the network as a struct variable then set the network value via the new function, mean we don't have to do this 

```
 let network_kind = match &USERNETWORK {
            Bitcoin => Network::Bitcoin,
            Testnet => Network::Testnet,
            _ => Network::Testnet,
        };

```
where we need it

Reply:
But if `USERNETWORK` is a `bitcoin::Network`, we don't need to do that either way. That's what I mean. And according to `main.rs`, it is a `Network`, not a string.